tests/unit/Switch-bidi.js

```diff
@@ -0,0 +1,136 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	var container, html = "<d-switch id='b1' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' textDir='ltr'></d-switch>" +
+		"<d-switch id='b2' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='false' textDir='rtl'></d-switch>" +
+		"<d-switch id='b3' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' title='ABC \u05d0\u05d1\u05d2' textDir='auto'></d-switch>";
+
+	registerSuite ({
+		name: "deliteful/Switch (bidi)",
+		setup: function () {
+			moduleRequire = require.config({
```

Comment:
note that this solution is actually not working, I had to comment out bidi  tests from the repo (see: c32a6d83ce00014628f45b6f8b0ff637efbe0c1f). The reason is that by using context require, modules likes Switch happened to be loaded twice. Once with bidi and once without. This sounds good. The problem is that then we end up registering the d-switch  custom element twice and this is something that is forbidden by custom element specification and fire exceptions at least in Chrome.